x-pack/docs/en/rest-api/security/get-api-keys.asciidoc

```diff
@@ -39,13 +39,20 @@ pertain to retrieving api keys:
 
 `realm_name`::
 (Optional, string) The name of an authentication realm. This parameter cannot be
-used with either `id` or `name`.
+used with either `id` or `name` or when `owner` flag is set to `true`.
 
 `username`::
 (Optional, string) The username of a user. This parameter cannot be used with
-either `id` or `name`.
+either `id` or `name` or when `owner` flag is set to `true`.
 
-NOTE: While all parameters are optional, at least one of them is required.
+`owner`::
+(Optional, boolean) A boolean flag that can be used to query API keys owned
+by the currently authenticated user. Defaults to false. If this parameter is set to
+`true` then `realm_name` or `username` parameters cannot be specified as
+they default to the authenticated users name and realm name.
```

Comment:
`as they are assumed to be the currently authenticated ones.` (don't use `default` in this case).